coretex/entities/annotation/image/coretex_format.py

```diff
@@ -154,7 +154,8 @@ def centroid(self) -> Tuple[int, int]:
             Tuple[int, int] -> x, y coordinates of centroid
         """
 
-        flattenedSegmentations = [element for sublist in self.segmentations for element in sublist]
+        flattenedSegmentations = [element for sublist in self.segmentations for i,  element in enumerate(sublist)\
+                                  if i < len(sublist) - 2]
```

Comment:
This was fixed in the annotation tool for coretex, so we no longer need to have the `if` part of the list comprehension